src/firestore/observable/fromRef.ts

```diff
@@ -1,8 +1,8 @@
-import { Observable, SchedulerLike } from 'rxjs';
+import { Observable, SchedulerLike, asyncScheduler } from 'rxjs';
 import { DocumentReference, Query, Action, Reference, DocumentSnapshot, QuerySnapshot } from '../interfaces';
 import { map, share } from 'rxjs/operators';
 
-function _fromRef<T, R>(ref: Reference<T>, scheduler?: SchedulerLike): Observable<R> {
+function _fromRef<T, R>(ref: Reference<T>, scheduler: SchedulerLike = asyncScheduler): Observable<R> {
```

Comment:
Match firestore behavior with rtdb behavior (same default scheduling). The scheduler is always specified to use the ZoneScheduler when using the normal Angularfire API, this is just to cover the possibility of someone using fromRef directly (to avoid a breaking change for this case).